src/lnp/application/message.rs

```diff
@@ -71,7 +71,7 @@ pub enum Messages {
     /// Within the received ping message, the sender will specify the number of
     /// bytes to be included within the data payload of the pong message.
     #[lnp_api(type = 19)]
-    #[display("pong(...)")]
+    #[display(Debug)]
```

Comment:
the size of the data may be large, and these data are just a random noise. So we need to leave the original version here in this specific case

```suggestion
    #[display("pong(...)")]
```

Reply:
Done. 

Parallel question, why would pong messages be very large? Aren't they suppose to be as small as possible?